helix-core/src/main/java/org/apache/helix/tools/ClusterVerifiers/ZkHelixClusterVerifier.java

```diff
@@ -48,6 +50,10 @@
   private static Logger LOG = LoggerFactory.getLogger(ZkHelixClusterVerifier.class);
   protected static int DEFAULT_TIMEOUT = 300 * 1000;
   protected static int DEFAULT_PERIOD = 500;
+  // COOL_DOWN before starting vefiyByPool
+  // The goal is to make sure waiting for controller pipeline starts at least one cycle
+  // to update ideal state.
+  protected static int DEFAULT_COOLDOWN = 2 * 1000;
```

Comment:
I think this is another workaround to avoid issue https://github.com/apache/helix/issues/526. If this is the case, then it would be an overkill to add to the general verifier logic. Because not all verifying logic depends on the pipeline running.